src/os_posix/os_fs.c

```diff
@@ -344,6 +344,14 @@ __posix_file_close(WT_FILE_HANDLE *file_handle, WT_SESSION *wt_session)
     session = (WT_SESSION_IMPL *)wt_session;
     pfh = (WT_FILE_HANDLE_POSIX *)file_handle;
 
+    if (S2C(session)->mmap) {
+        __wt_verbose(
+          session, WT_VERB_FILEOPS, "%s, file-close: fd=%d\n", file_handle->name, pfh->fd);
```

Comment:
Why is this verbose message inside the mmap test?

Only flagging this once, there are other examples.